docs/types.rst

```diff
@@ -163,25 +174,30 @@ Operators:
 .. note::
     Starting with version 0.5.0 contracts do not derive from the address type, but can still be explicitly converted to
     ``address`` or to ``address payable``, if they have a payable fallback function.
+    The distinction between ``address`` and ``address payable`` was introduced with version 0.5.0.
```

Comment:
The previous sentence kind of says this too? Perhaps reword the two sentences into one?

Reply:
Yep, will merge the sentences.

Reply:
Hm, didn't really work, but at least I swapped them.